packages/scandipwa/src/component/MyAccountInformation/MyAccountInformation.container.js

```diff
@@ -137,38 +144,47 @@ export class MyAccountInformationContainer extends PureComponent {
         updateCustomerLoadingStatus(true);
 
         await this.handleInformationChange({ firstname, lastname, taxvat });
+        const { isLocked } = this.state;
```

Comment:
If user isLocked shouldn't he be logged out of MyAccount, thus making this `isLocked` obsolete?